modules/osdk-cli-reference-run.adoc

```diff
@@ -7,54 +7,56 @@
 
 The `operator-sdk run` command provides options that can launch the Operator in various environments.
 
-.`run` arguments
+[id="osdk-cli-reference-run-bundle_{context}"]
+== bundle
+
+The `run bundle` subcommand deploys an Operator in the bundle format with Operator Lifecycle Manager (OLM).
+
+.`run bundle` flags
 [options="header",cols="1,3"]
 |===
-|Arguments |Description
+|Flag |Description
 
-|`--kubeconfig` (string)
-|The file path to a Kubernetes configuration file. Default: `$HOME/.kube/config`
+|`--index-image` (string)
+|Index image in which to inject a bundle. This default is `quay.io/operator-framework/upstream-opm-builder:latest`.
 
-|`--local`
-|The Operator is run locally by building the Operator binary with the ability to access a Kubernetes cluster using a `kubeconfig` file.
+|`--install-mode <install_mode_value>`
+|Install mode
```

Comment:
Which values are available?

Reply:
Update with two examples and link to the rest.